app/Http/Controllers/Backend/BorrowController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use Illuminate\Http\Request;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Borrow;
+use App\BorrowDetail;
+
+class BorrowController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $borrows = Borrow::all();
+        return view('admin.borrows.index', compact('borrows'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store()
+    {
+        //
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param int $id Borrow
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        try {
+            $borrowDetail = BorrowDetail::where('borrow_id', $id)->get();
```

Comment:
[rank A] Are you using findOrFail function for other? You can write this as:
$borrowDetail = BorrowDetail::where('borrow_id', '=', $id)->firstOrFail();